creator-node/test/StateMonitoringManager.test.js

```diff
@@ -0,0 +1,271 @@
+/* eslint-disable no-unused-expressions */
+const nock = require('nock')
+const chai = require('chai')
+const sinon = require('sinon')
+const { expect } = chai
+chai.use(require('sinon-chai'))
+chai.use(require('chai-as-promised'))
```

Comment:
can we set these in a `before` block? would prefer that the top of file is for imports only and as little mutations as possible

Reply:
hmm I think standard practice is to put it at the top of the file - what about moving it to right below the last import?